packages/adblocker-electron/content.ts

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Copyright (c) 2017-2019 Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import { ipcRenderer, remote, webFrame } from 'electron';
+
+import { IBackgroundCallback, IMessageFromBackground } from '../content/communication';
```

Comment:
```js
import {
  IBackgroundCallback,
  IMessageFromBackground,
} from '@cliqz/adblocker-webextension-cosmetics';
```

You might need to add `@cliqz/adblocker-webextension-cosmetics`. Let's check if this works then I can refactor it to drop the `webextension` in the name.

Reply:
It built only adblocker.js with the following typescript error:
```
../../tsconfig.json:25:5 - error TS5023: Unknown compiler option 'strictBindCallApply'.

25     "strictBindCallApply": true,
```